vscode/test/suites/language-service/language-service.test.ts

```diff
@@ -204,12 +217,11 @@ suite("Q# Language Service Tests", function suite() {
   });
 
   test("Package dependencies", async () => {
-    const doc = await vscode.workspace.openTextDocument(projectMainDocUri);
-    vscode.commands.executeCommand("workbench.action.problems.focus");
+    const doc = await vscode.workspace.openTextDocument(mainPackageMainQs);
```

Comment:
If the removed `executeCommand("workbench.action.problems.focus")` wasn't needed here, is it still needed in the test case below (line 262)?